docs/build-tools/maven.md

```diff
@@ -27,32 +31,24 @@ The Bloop plugin for Maven doesn't need to be installed, you can run
 following command.
 
 ```bash
-$ mvn ch.epfl.scala:maven-bloop_2.13:@VERSION@:bloopInstall
+$ mvn ch.epfl.scala:bloop-maven-plugin:@BLOOP_MAVEN_VERSION@:bloopInstall
 ```
 
 In some cases, when you are using generated sources in you project
 you might need to run:
 
 ```bash
-$ mvn generate-sources ch.epfl.scala:maven-bloop_2.13:@VERSION@:bloopInstall
-```
-
-Note that the following command uses the latest Bloop stable version
-@VERSION@.
-
-Here is the list of the latest Bloop stable and development versions. If you
-want to be on the very latest at the cost of misbehaviors, replace the above
-question with the development version.
-
-```scala mdoc:releases
-I am going to be replaced by the docs infrastructure.
+$ mvn generate-sources ch.epfl.scala:bloop-maven-plugin:@BLOOP_MAVEN_VERSION@:bloopInstall
 ```
 
+**Note**: if you've used this command before you'll have noticed the artifact
+name has changed. In the past you would have used `maven-bloop_2.13`, however as
+of the 2.x series the artifact name as changed to `bloop-maven-plugin`.
 
 To build with a single Scala project `foo` generates two configuration files:
 
 ```bash
-$ mvn ch.epfl.scala:maven-bloop_2.10:@VERSION@:bloopInstall
+$ mvn ch.epfl.scala:bloop-maven-plugin:@BLOOP_MAVEN_VERSION@:bloopInstall
```

Comment:
Just out of curiosity, what is the Scala version of the Bloop Maven plugin? Can it conflict with other Maven plugins compiled with Scala?

Reply:
So the Scala version used for the plugin is 2.13. And honestly I'm not 100% sure how that works with Maven. I'm not sure if the same classpath for the build is shared with the plugin goals or not, so I'm not sure if there will be a conflict. I guess I didn't even think about that when I was testing on various repos.

It's currently only being published for 2.13 in this repo as well, so I didn't change that and instead just kept the behavior the same.

Reply:
It seems that the Scala Maven Plugin also [depends on Scala 2.13](https://github.com/davidB/scala-maven-plugin/blob/b64ca9c868466fb741dd05fc16e024c34d021d02/pom.xml#L184-L186) so that's probably why.